README.md

```diff
@@ -7,59 +7,411 @@
+    // they're notified of on the settlement ledger.
+    const txid = Network.makePaymentToPeer(diff)
+
+    return { txid }
+  },
+
+  handleIncomingClaim: async function (ctx, claim) {
+    const { txid } = claim
+    const payment = await Network.getPayment(txid)
+
+    if (!payment) {
+      return
+    }
+
+    // It doesn't really matter whether this is fulfilled or not, we just need
+    // it to affect the incoming balance so we know how much has been received.
+    // We use the txid as the ID of the incoming payment, so it's impossible to
+    // apply the same incoming settlement transfer twice.
+    await ctx.state.incomingSettlements.prepare({
+      id: txid,
+      amount: payment.amount
+    }, true) // isIncoming: true
+  }
+
+}
 ```
+
+## Backend API, with Extensions for Payment Channels
+
+-------
+
+### `async TransferLog.getIncomingFulfilled ()`
+
+Get the sum of all incoming payments in the `fulfilled` state.
+
+#### Returns
+
+- `return` (Integer String) incoming balance.
+
+-------
+
+### `async TransferLog.getIncomingFulfilledAndPrepared ()`
+
+Get the sum of all incoming payments, including those which are in the `prepared` state.
+
+#### Returns
+
+- `return` (Integer String) highest incoming balance.
+
+-------
+
+### `async TransferLog.getOutgoingFulfilled ()`
+
+Get the sum of all outgoing payments in the `fulfilled` state.
+
+#### Returns
+
+- `return` (Integer String) outgoing balance.
+
+-------
+
+### `async TransferLog.getOutgoingFulfilledAndPrepared ()`
+
+Get the sum of all outgoing payments, including those which are in the `prepared` state.
+
+#### Returns
+
+- `return` (Integer String) highest outgoing balance.
+
+-------
+
+### `getMaxValueTracker (key)`
+
+Get a MaxValueTracker.
+
+#### Parameters
+
+- `key` (String) name of this value tracker.
+
+#### Returns
+
+- `return` (MaxValueTracker) max value tracker.
+
+-------
+
+### `async MaxValueTracker.setIfMax (entry)`
+
+Put `entry` into the MaxValueTracker. If `entry.value` is larger than the
+previous entry's value, then `entry` becomes the new max entry, and the
+previous max entry is returned. If `entry.value` is not larger than the
+previous max entry's value, then the max entry remains the same and `entry` is
+returned back.
+
+#### Parameters
+
+- `entry` (Object) entry to add to the max value tracker.
+- `entry.value` (Integer String) value to compare to the current max value.
+- `entry.data` (Object) data to attach to the entry.
+
+#### Return
+
+- `return` (Object) previous max entry or `entry`.
+- `return.value` (Integer String) value of returned entry.
+- `return.data` (Object) data attached to returned entry.
+
+-------
+
+### `async MaxValueTracker.getMax ()`
+
+Returns the max value tracker's maximum entry.
+
+#### Return
+
+- `return` (Object) max entry of the max value tracker.
+- `return.value` (Integer String) value of returned entry.
+- `return.data` (Object) data attached to returned entry.
+
+## Payment Channel Backend API
```

Comment:
What about Payment Channel Module API, or Payment Channel Plugin API? I'm kinda leaning towards Payment Channel Plugin API because it's the API that you have to implement for every payment channel plugin. It's also a bit more of a plugin than it is a backend. It might be confusing to have two nested types of plugins though: a Ledger Plugin with a Payment Channel Plugin inside.